test/integration/targets/win_psrepository/defaults/main.yml

```diff
@@ -1,4 +1,25 @@
 ---
 repository_name: My Get
-repository_sourcelocation: https://www.myget.org/F/powershellgetdemo/api/v2
-repository_sourcelocation2: '{{ remote_tmp_dir }}'
+repository_source_location: https://www.myget.org/F/powershellgetdemo/api/v2
+repository_publish_location: '{{ redirect_publish_expected_target }}'
+repository_script_source_location: '{{ redirect_script_source_expected_target }}'
+repository_script_publish_location: '{{ redirect_script_publish_expected_target }}'
+
+repository_source_location2: '{{ remote_tmp_dir }}'
+
+redirect_url:                             http://httpbin.org/redirect
```

Comment:
httpbin.org is notoriously temperamental in CI and causes all sorts of unreachable errors when we run it. To combat this issue we have a test container that runs the httpbin code locally. What you need to do to access this is

* Add the `needs/httptester` in your test target alias like you can see with the [win_get_url](https://github.com/ansible/ansible/blob/devel/test/integration/targets/win_get_url/aliases) aliases (you don't need to skip windows 2008 as we no longer test that version)
* Add `prepare_http_tests` in the test target meta/main.yml. This sets things up like certificates and gets it ready for cleanup once the test is finished
* In your tests/variables change any reference `http://httbin.org/<path>` to `http://{{ httpbin_host }}/<path>`. This is automatically set to the private instance if available or just `httpbin.org` if httptester is not available

The beauty of httptester is that it is reliable, a private instance, has trusted https certs, and if ansible-test can't set it up the `{{ httpbin_host }}` var just points to the public instance for your own testing.

Reply:
Ah, that's fantastic! Really appreciate the detailed instructions there.